.changeset/popular-camels-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+"crib-deploy-environment": minor
```

Comment:
shouldn't this be a major bump? because we're introducing a breaking change, by requesting two new inputs

Reply:
Makes sense. Fixed.